src/Reflection/ReflectionClass.php

```diff
@@ -1028,4 +1028,12 @@ function (Node\Name $interfaceName) {
             )
         );
     }
+
+    /**
+     * @throws Exception\Uncloneable
+     */
+    public function __clone()
+    {
+        throw new Exception\Uncloneable('Trying to clone an uncloneable object of class ' . __CLASS__);
```

Comment:
Each of these strings are duplicate: instead, create a public static constructor on `Uncloneable` and call it from all around the various `__clone` methods